### PR TITLE
Screen forced ON while activating or deactivating Omnipod Dash

### PR DESCRIPTION
**Pull request by olorinmaia**
This will solve feature request in: https://github.com/nightscout/AndroidAPS/issues/2896

### CODE REVIEW

File: pump/omnipod-common/src/main/java/info/nightscout/androidaps/plugins/pump/omnipod/common/ui/wizard/deactivation/PodDeactivationWizardActivity.kt [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
 import android.os.Bundle
+import android.view.WindowManager
 import info.nightscout.androidaps.plugins.pump.omnipod.common.R
 import info.nightscout.androidaps.plugins.pump.omnipod.common.ui.wizard.common.activity.OmnipodWizardActivityBase
 
 abstract class PodDeactivationWizardActivity : OmnipodWizardActivityBase() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        window.addFlags(WindowManager.LayoutParams.FLAG_KEEP_SCREEN_ON)
```
Comment (jbr7rr) [inline pump/omnipod-common/src/main/java/info/nightscout/androidaps/plugins/pump/omnipod/common/ui/wizard/deactivation/PodDeactivationWizardActivity.kt:12]: Should not be needed here as this class inherent from OmnipodWizardActivityBase in which you made the same change

Comment (olorinmaia) [reply pump/omnipod-common/src/main/java/info/nightscout/androidaps/plugins/pump/omnipod/common/ui/wizard/deactivation/PodDeactivationWizardActivity.kt]: fixed

**Comment by MilosKozak** [comment]
anyone tested it?

**Comment by olorinmaia** [comment]
I have tested first step of activating and deactivating with Dash and screen stays permanent lit there. I haven't been through all steps as I'm waiting for my pod to expire in my testrig. But I assume it works for all steps. If not I can test this also tomorrow.

**Comment by olorinmaia** [comment]
> anyone tested it?

@MilosKozak  Testet OK now with Omnipod Dash, bot activate and deactivate. All steps.

**Comment by jakobheyman** [comment]
> anyone tested it?

I have also tested it and it worked good with both deactivate and activate.

**Pull request merged**